prlworkflows/sqs.py

```diff
@@ -301,18 +296,20 @@ def enumerate_sqs(structure, subl_model, endmembers=True, scale_volume=True, ski
     # error checking
```

Comment:
Probably can drop this comment as well, since our error checking is much more simple now that we handle lower order SQS properly.